src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_params.py

```diff
@@ -36,8 +36,11 @@ def get_device_id_completion_list(prefix, action, parsed_args, **kwargs):#pylint
 # Arguments for 'iot hub policy' group
 register_cli_argument('iot hub policy', 'policy_name', options_list=('--name', '-n'), help='Share access policy name.')
 
+# Arguments for 'iot hub job' group
+register_cli_argument('iot hub job', 'job_id', help='Job Id.')
+
 # Arguments for 'iot hub create'
-register_cli_argument('iot hub create', 'hub_name', options_list=('--name', '-n'), completer=None)
+register_cli_argument('iot hub create', 'hub_name', completer=None)
```

Comment:
@xscript I'll create an issue to track this and that way we can merge your PR. It looks like I owe some guidance in our command authoring guide to cover this. anyways.